stf.test/META-INF/MANIFEST.MF

```diff
@@ -7,6 +7,7 @@ Bundle-Vendor: Saros Project
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: saros.eclipse,
  saros.core,
+ saros.libratory,
```

Comment:
This should not be necessary. The core already includes and exports them. The worst thing that can happen in an OSGi context is that this plugin will see a different "set" of the libraries than the core. i.e the code is loaded twice into the VM and every static variable will be seen different depending which class loader loaded which class.

Reply:
But we did not include the saros.libratory package in the `Export-Package`-section of the core's MANIFEST.MF. Would it be exported anyway?
We thought it would be preferable (to reduce coupling) if the core would not export the libratory package transitively.